<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="sbInicio">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbDuracion">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Manualmente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSimularDesbordamientoControlado">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>570</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Desbordamiento Controlado</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gvMemoria">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>831</width>
      <height>581</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblProcesos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnComenzar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Comenzar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblProcesosSimulados">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbCantidad">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerarAleatorios">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>20</width>
      <height>631</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSimularDesbordamientoNoControlado">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>610</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Desbordamiento no Controlado</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Orden de ejecución</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Duración:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MEMORIA DE PROCESOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Datos de los procesos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aleatorio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SEGURIDAD EN LA GESTION DE PROCESOS</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>831</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>40</y>
      <width>20</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>123</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>630</y>
      <width>831</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
